src/checks/incorrect_delimiter.rs

```diff
@@ -126,21 +132,6 @@ mod tests {
         assert_eq!(None, checker.run(&line));
     }
 
-    #[test]
-    fn leading_space_run() {
-        let mut checker = IncorrectDelimiterChecker::default();
-        let line = LineEntry {
-            number: 1,
-            file: FileEntry {
-                path: PathBuf::from(".env"),
-                file_name: ".env".to_string(),
-                total_lines: 1,
-            },
-            raw_string: String::from(" FOO=FOOBAR"),
```

Comment:
It would be nice if you wrote a test for the case with `*FOO=BAR`.

Reply:
No problem, I fixed here: https://github.com/dotenv-linter/dotenv-linter/pull/259/commits/49d2a79a5c07495eaf14372f57dc844889e958f9